avm/src/vm/opcode.rs

```diff
@@ -3037,12 +3065,16 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
       let recurse_fn = HandlerFragment::new(slf.read_fixed(1), 0);
       let (mut seq, _) = hm.read_from_fractal(&slf, 0);
       let curr = seq.read_fixed(0);
+      let mut hand_mem;
       if curr < seq.read_fixed(1) {
         hm.write_fixed_in_fractal(&mut seq, 0, curr + 1);
         hm = recurse_fn.run(hm).await;
-        hand_mem.join(hm);
+        hm.drop_parent();
+        hand_mem = Arc::try_unwrap(hand_mem_ref).unwrap();
```

Comment:
^

Reply:
left this one out, since there was only one reference created from the arc